src/main/scala/com/github/athieriot/CleanAfterExample.scala

```diff
@@ -1,16 +1,24 @@
 package com.github.athieriot
 
-import scala.collection.JavaConversions._
-import com.mongodb.{ MongoClient, ServerAddress }
-import org.specs2.specification.AfterExample
+import org.specs2.specification.AfterEach
+import reactivemongo.api.MongoDriver
 
-trait CleanAfterExample extends AfterExample {
+import scala.concurrent.Await
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration._
+
+trait CleanAfterExample extends AfterEach {
   self: EmbedConnection =>
 
-  lazy val mongoClient = new MongoClient(new ServerAddress(network.getServerAddress(), network.getPort()));
+  def getConn = {
+    val driver = new MongoDriver
+    val db = driver.connection(List(s"127.0.0.1:${network.getPort}"))("testdb")
```

Comment:
Calling "apply" is [deprecated](https://github.com/ReactiveMongo/ReactiveMongo/blob/master/driver/src/main/scala/api/api.scala#L301), better use "database" method.
Hardcoding the name "testdb" is creating a restriction on database name. It seems like there's no way of getting database names in ReactiveMongo, I couldn't find any API through brief googling. Maybe we should use MongoClient here.